### ISSUE TITLE
Unable to ping http://ascoderu.ca/lokole/ using legitimate GitHub assigned token

### ISSUE THREAD

**Issue by nzola** [ascoderu/lokole#599]
I am not able to ping Lokole main mail server [http://ascoderu.ca/lokole/](url) dashboard hosted in Azure. This prevents to see Dashboard status and mail transactions between the server and the clients (Box Lokoles). 
Note that the server is running, but the communications part is broken.
![image](https://user-images.githubusercontent.com/17815091/197932101-5a561e24-7524-435b-b153-d980bf99d828.png)

**Comment by mabuelhagag** [comment]
While opening the [ping endpoint](https://mailserver.lokole.ca/healthcheck/ping) looks like it's an SSL certificate issue.

**Comment by nzola** [comment]
# Ok. We hadcthis problème before and you fixed it.

**Comment by holta** [comment]
> While opening the [ping endpoint](https://mailserver.lokole.ca/healthcheck/ping) looks like it's an SSL certificate issue.

Indeed.  Opening any browser to https://mailserver.lokole.ca reveals that Lokole allowed this (paid or unpaid) certificate to lapse:

"This server could not prove that it is mailserver.lokole.ca; its security certificate expired 43 days ago."

**Comment by nzola** [comment]
How do I renew the certificate

**Comment by holta** [comment]
This is a business matter:

Clemens Wolff or others hopefully provided you a sheet of passwords and accounts, outlining what's needed for regular upkeep.

**Comment by nzola** [comment]
> This is a business matter:
> 
> Clemens Wolff or others hopefully provided you a sheet of passwords and accounts, outlining what's needed for regular upkeep.

Yes. Clemens had provided me with all the passwords

**Comment by nzola** [comment]
> I am not able to ping Lokole main mail server [http://ascoderu.ca/lokole/](url) dashboard hosted in Azure. This prevents to see Dashboard status and mail transactions between the server and the clients (Box Lokoles). Note that the server is running, but the communications part is broken. ![image](https://user-images.githubusercontent.com/17815091/197932101-5a561e24-7524-435b-b153-d980bf99d828.png)

This issue was resolved last week. I rebooted the main server in Microsoft Azure cloud and everything returns normal. Now I can ping the Lokole VM and mail transfers from/to internet is working smoothly.
